BroView.cpp

```diff
@@ -818,6 +818,10 @@ void CChildView::ResizeWindowPopupInpl()
 			//初期値(アクティブウインドウの幅)
 			rcFeature.right = rcCurrent.Width();
 
+			//タスクバーを除く画面サイズ
+			RECT rcDisp = {0};
+			SystemParametersInfo(SPI_GETWORKAREA, NULL, &rcDisp, NULL);

```

Comment:
第2、第4引数は`UINT`なので`0`の方が良いかな。

```suggestion
			SystemParametersInfo(SPI_GETWORKAREA, 0, &rcDisp, 0);

```

Reply:
ありがとうございます。適用しました。